lib/config/defaults.ts

```diff
@@ -23,7 +23,7 @@ export function getConfig(): AllConfig {
   const options = getOptions();
   const config: AllConfig = {};
   options.forEach((option) => {
-    if (!option.parents) {
+    if (!option.parents || option.parents.includes('.')) {
```

Comment:
This would make `default` noop for all root options. That is if we add the parents as expected.

Reply:
Sry misread here...